src/api/hydration.ts

```diff
@@ -40,7 +40,7 @@ export const getSchema_Resource = async (connectorId: string | null) => {
     return resourceSchema;
 };
 
-const liveSpecColumns = `id,spec_type,spec,writes_to,reads_from,last_pub_id`;
+const liveSpecColumns = `id,spec_type,spec,writes_to,reads_from,last_pub_id,updated_at`;
```

Comment:
These hydration services should be ported over to their corresponding Supabase table named file in the _src/api/_ directory soon. It would be _best_ to do this in a separate PR given the number of changes made in this one.